<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenPoseSquat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="m_strimport">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnimport">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>941</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Result</string>
    </property>
    <widget class="QLabel" name="m_lblimage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>901</width>
       <height>381</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="m_btnstart">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>710</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OpenPose Squat</string>
    </property>
   </widget>
  </widget>
    <widget class="QLabel" name="m_lbllivefeed">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(To use LiveFeed, press 'Start' without importing a video file)</string>
    </property>
   </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
